artery.gemspec

```diff
@@ -27,5 +27,5 @@ Gem::Specification.new do |s|
 
   s.add_dependency 'nats',         '~> 0.8'
```

Comment:
Maybe here we should tighten upper bound, too?

Reply:
It's better not to lock patch version, let's do `< 0.12`.

Reply:
done